students/LuisGonzalez/session06/trapz.py

```diff
@@ -0,0 +1,11 @@
+def trapz(fun, a, b, N, **kwargs):
```

Comment:
Another way to think of it: use positionals for required arguments and keyword for optional arguments.